html5/index.html

```diff
@@ -1379,6 +1379,17 @@ <h2>Xpra Bug Report</h2>
 						client._poll_clipboard(e);
 					}
 				});
+
+				// Prevent stuck keys.
+				$(window).on("blur", (e) => {
+					if (client.last_key_packet.length > 2 && client.last_keycode_pressed > 0) {
+						console.log("clearing keycode: " + client.last_keycode_pressed);
+						key_packet = client.last_key_packet;
+						// Set pressed = false to indicate key up.
+						key_packet[3] = false;
+						client.send(key_packet);
+					}
+				});
```

Comment:
There is no `last_key_packet` in the current client, did you miss a chunk?

Shouldn't this also update `last_key_packet` after sending it? (to prevent multiple unpresses - those won't do anything, but this would be more correct and potentially prevent problems when sharing is enabled)

Reply:
Yes, there is a chunk missing from the commit that patches Client.js. I'll push the missing changes.